content/yaml-testing/testing.md

```diff
@@ -46,6 +46,15 @@ scripts:
     test_report: test-results/flutter.json
 ```
 
+If running Flutter integration tests from yaml fails because the build doesn't wait for the emulator to be launched, then adding **adb wait-for-device** command could fix the issue:
```

Comment:
Just wondering if this command should be included in the example below by default?

Reply:
It was added to the default script in React Native integration test section on the same page. I believe we can also add it to the Flutter the same way rather than making it as a separate paragraph as I did in the PR.

Reply:
For example:
flutter emulators --launch apple_ios_simulator  &        # for android use: flutter emulators --launch emulator
adb wait-for-device